src/gwt/src/org/rstudio/studio/client/workbench/prefs/views/PackagesPreferencesPane.java

```diff
@@ -364,6 +348,7 @@ public RestartRequirement onApply(UserPrefs prefs)
       String mirrorTextValue = cranMirrorTextBox_.getTextBox().getText();
 
       boolean cranRepoChanged = !mirrorTextValue.equals(cranMirrorStored_);
+      // TODO: Is this correct?  Custom doesn't require a leading http://, so this wont always match
```

Comment:
We don't usually leave TODOs in the code, although you're right that there could be an issue here (please file it separately!)